docs/how-to/code/convert-to-pebble-layer/rockcraft.yaml

```diff
@@ -1,9 +1,9 @@
 name: custom-nginx-rock
 base: "ubuntu@22.04"
 version: latest
-summary: An NGINX ROCK
+summary: An NGINX rock
```

Comment:
Not the focus here, but upstream nginx call themselves "nginx" in lowercase.

Reply:
good point, but is it though? I've seen the 3 forms :p NGINX, Nginx and nginx xD

Reply:
You're right, the official spelling is totally inconsistent.